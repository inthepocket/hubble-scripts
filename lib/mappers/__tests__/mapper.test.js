const FIXTURES = require('./__fixtures__/mapper');

const mapper = require('..');
const formatColor = require('../color');

describe('[mapper.js] test suite', () => {
  describe('-> #formatColor:', () => {
    it('given RGB NSColor, should transform it to a nicely parsed object', () => {
      const [ULTRAMARINE] = FIXTURES.DOCUMENT_COLORS;
      expect(formatColor(ULTRAMARINE)).toMatchSnapshot();
    });
  });

  describe('-> #mapColors:', () => {
    it('given the document colors, should transform all colors nicely', () => {
      expect(mapper.mapColors(FIXTURES.DOCUMENT_COLORS)).toMatchSnapshot();
    });

    it('given artboard colors, should transform all colors nicely', () => {
      expect(mapper.mapColors(FIXTURES.ARTBOARD_COLORS)).toMatchSnapshot();
    });
  });

  describe('-> #mapTextStyles:', () => {
    it('given the text styles, should transform them nicely', () => {
      const parsed = mapper.mapTextStyles(FIXTURES.TEXT_STYLES);
      expect(parsed).toMatchSnapshot();

      const [heading1, heading2, paragraph] = parsed;
      expect(heading1.id).toBe('Heading 1');
      expect(heading2.id).toBe('Heading 2');
      expect(paragraph.id).toBe('Paragraph');
    });
  });

  describe('-> #mapGradients:', () => {
    it('given artboard gradients, should transform them nicely', () => {
      expect(mapper.mapGradients(FIXTURES.ARTBOARD_GRADIENTS)).toMatchSnapshot();
    });

    it('given docuement gradients, should transform them nicely', () => {
      expect(mapper.mapGradients(FIXTURES.DOCUMENT_GRADIENTS)).toMatchSnapshot();
    });
  });

  describe('-> #mapShadows:', () => {
    it('given artboard gradients, should transform them nicely', () => {
      const parsed = mapper.mapShadows(FIXTURES.ARTBOARD_SHADOWS);
      expect(parsed).toMatchSnapshot();
    });
  });
});
