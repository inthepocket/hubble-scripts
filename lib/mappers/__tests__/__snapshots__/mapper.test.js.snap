// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mapper.js] test suite -> #formatColor: given RGB NSColor, should transform it to a nicely parsed object 1`] = `
Object {
  "alpha": 1,
  "blue": 159,
  "green": 6,
  "hex": "10069f",
  "hsl": Array [
    244,
    93,
    32,
  ],
  "name": "ultramarine",
  "red": 16,
  "rgb": Array [
    16,
    6,
    159,
  ],
  "rgba": Array [
    16,
    6,
    159,
    1,
  ],
  "token": "color",
}
`;

exports[`[mapper.js] test suite -> #mapBorders: given artboard borders, should transform them nicely 1`] = `
Array [
  Object {
    "color": Object {
      "alpha": 1,
      "blue": 150,
      "green": 150,
      "hex": "ff9696",
      "hsl": Array [
        0,
        100,
        79,
      ],
      "name": "geraldine",
      "red": 255,
      "rgb": Array [
        255,
        150,
        150,
      ],
      "rgba": Array [
        255,
        150,
        150,
        1,
      ],
      "token": "color",
    },
    "id": "inside",
    "radius": 3,
    "token": "border",
    "type": "INSIDE",
    "width": 1,
  },
  Object {
    "color": Object {
      "alpha": 1,
      "blue": 150,
      "green": 150,
      "hex": "ff9696",
      "hsl": Array [
        0,
        100,
        79,
      ],
      "name": "geraldine",
      "red": 255,
      "rgb": Array [
        255,
        150,
        150,
      ],
      "rgba": Array [
        255,
        150,
        150,
        1,
      ],
      "token": "color",
    },
    "id": "outside",
    "radius": 12.5,
    "token": "border",
    "type": "OUTSIDE",
    "width": 1,
  },
  Object {
    "gradient": Object {
      "from": Array [
        0.5,
        1,
      ],
      "stops": Array [
        Object {
          "color": Object {
            "alpha": 1,
            "blue": 144,
            "green": 182,
            "hex": "97b690",
            "hsl": Array [
              109,
              21,
              64,
            ],
            "name": "norway",
            "red": 151,
            "rgb": Array [
              151,
              182,
              144,
            ],
            "rgba": Array [
              151,
              182,
              144,
              1,
            ],
            "token": "color",
          },
          "position": 0,
        },
        Object {
          "color": Object {
            "alpha": 1,
            "blue": 88,
            "green": 145,
            "hex": "559158",
            "hsl": Array [
              123,
              26,
              45,
            ],
            "name": "hippieGreen",
            "red": 85,
            "rgb": Array [
              85,
              145,
              88,
            ],
            "rgba": Array [
              85,
              145,
              88,
              1,
            ],
            "token": "color",
          },
          "position": 0.5038507579143953,
        },
        Object {
          "color": Object {
            "alpha": 1,
            "blue": 62,
            "green": 125,
            "hex": "227d3e",
            "hsl": Array [
              138,
              57,
              31,
            ],
            "name": "salem",
            "red": 34,
            "rgb": Array [
              34,
              125,
              62,
            ],
            "rgba": Array [
              34,
              125,
              62,
              1,
            ],
            "token": "color",
          },
          "position": 1,
        },
      ],
      "to": Array [
        0.4999999999999999,
        3.0616169978683836e-17,
      ],
      "token": "gradient",
      "type": "LINEAR",
    },
    "id": "gradientWithCorners",
    "radius": Array [
      8,
      12.5,
      12.5,
      12.5,
    ],
    "token": "border",
    "type": "OUTSIDE",
    "width": 1,
  },
  Object {
    "color": Object {
      "alpha": 1,
      "blue": 150,
      "green": 150,
      "hex": "ff9696",
      "hsl": Array [
        0,
        100,
        79,
      ],
      "name": "geraldine",
      "red": 255,
      "rgb": Array [
        255,
        150,
        150,
      ],
      "rgba": Array [
        255,
        150,
        150,
        1,
      ],
      "token": "color",
    },
    "id": "center",
    "radius": 0,
    "token": "border",
    "type": "CENTER",
    "width": 1,
  },
]
`;

exports[`[mapper.js] test suite -> #mapColors: given artboard colors, should transform all colors nicely 1`] = `
Array [
  Object {
    "alpha": 1,
    "blue": 230,
    "green": 194,
    "hex": "5fc2e6",
    "hsl": Array [
      196,
      73,
      64,
    ],
    "id": "primary900",
    "name": "seagull",
    "red": 95,
    "rgb": Array [
      95,
      194,
      230,
    ],
    "rgba": Array [
      95,
      194,
      230,
      1,
    ],
    "token": "color",
    "variant": 900,
  },
  Object {
    "alpha": 1,
    "blue": 209,
    "green": 193,
    "hex": "5ec1d1",
    "hsl": Array [
      188,
      56,
      59,
    ],
    "id": "primary800",
    "name": "viking",
    "red": 94,
    "rgb": Array [
      94,
      193,
      209,
    ],
    "rgba": Array [
      94,
      193,
      209,
      1,
    ],
    "token": "color",
    "variant": 800,
  },
  Object {
    "alpha": 1,
    "blue": 214,
    "green": 200,
    "hex": "77c8d6",
    "hsl": Array [
      189,
      54,
      65,
    ],
    "id": "primary700",
    "name": "viking",
    "red": 119,
    "rgb": Array [
      119,
      200,
      214,
    ],
    "rgba": Array [
      119,
      200,
      214,
      1,
    ],
    "token": "color",
    "variant": 700,
  },
  Object {
    "alpha": 1,
    "blue": 219,
    "green": 207,
    "hex": "8dcfdb",
    "hsl": Array [
      189,
      52,
      71,
    ],
    "id": "primary600",
    "name": "halfBaked",
    "red": 141,
    "rgb": Array [
      141,
      207,
      219,
    ],
    "rgba": Array [
      141,
      207,
      219,
      1,
    ],
    "token": "color",
    "variant": 600,
  },
  Object {
    "alpha": 1,
    "blue": 224,
    "green": 214,
    "hex": "a0d6e0",
    "hsl": Array [
      189,
      51,
      75,
    ],
    "id": "primary500",
    "name": "powderBlue",
    "red": 160,
    "rgb": Array [
      160,
      214,
      224,
    ],
    "rgba": Array [
      160,
      214,
      224,
      1,
    ],
    "token": "color",
    "variant": 500,
  },
  Object {
    "alpha": 1,
    "blue": 230,
    "green": 221,
    "hex": "b3dde6",
    "hsl": Array [
      191,
      50,
      80,
    ],
    "id": "primary400",
    "name": "powderBlue",
    "red": 179,
    "rgb": Array [
      179,
      221,
      230,
    ],
    "rgba": Array [
      179,
      221,
      230,
      1,
    ],
    "token": "color",
    "variant": 400,
  },
  Object {
    "alpha": 1,
    "blue": 240,
    "green": 235,
    "hex": "d4ebf0",
    "hsl": Array [
      191,
      48,
      89,
    ],
    "id": "primary300",
    "name": "pattensBlue",
    "red": 212,
    "rgb": Array [
      212,
      235,
      240,
    ],
    "rgba": Array [
      212,
      235,
      240,
      1,
    ],
    "token": "color",
    "variant": 300,
  },
  Object {
    "alpha": 1,
    "blue": 250,
    "green": 243,
    "hex": "e5f3fa",
    "hsl": Array [
      200,
      68,
      94,
    ],
    "id": "primary200",
    "name": "solitude",
    "red": 229,
    "rgb": Array [
      229,
      243,
      250,
    ],
    "rgba": Array [
      229,
      243,
      250,
      1,
    ],
    "token": "color",
    "variant": 200,
  },
  Object {
    "alpha": 1,
    "blue": 253,
    "green": 249,
    "hex": "f2f9fd",
    "hsl": Array [
      202,
      73,
      97,
    ],
    "id": "primary100",
    "name": "aliceBlue",
    "red": 242,
    "rgb": Array [
      242,
      249,
      253,
    ],
    "rgba": Array [
      242,
      249,
      253,
      1,
    ],
    "token": "color",
    "variant": 100,
  },
  Object {
    "alpha": 1,
    "blue": 62,
    "green": 125,
    "hex": "227d3e",
    "hsl": Array [
      138,
      57,
      31,
    ],
    "id": "secondary900",
    "name": "salem",
    "red": 34,
    "rgb": Array [
      34,
      125,
      62,
    ],
    "rgba": Array [
      34,
      125,
      62,
      1,
    ],
    "token": "color",
    "variant": 900,
  },
  Object {
    "alpha": 1,
    "blue": 94,
    "green": 94,
    "hex": "f55e5e",
    "hsl": Array [
      0,
      88,
      66,
    ],
    "id": "red",
    "name": "carnation",
    "red": 245,
    "rgb": Array [
      245,
      94,
      94,
    ],
    "rgba": Array [
      245,
      94,
      94,
      1,
    ],
    "token": "color",
    "variant": null,
  },
  Object {
    "alpha": 1,
    "blue": 74,
    "green": 135,
    "hex": "3e874a",
    "hsl": Array [
      130,
      37,
      39,
    ],
    "id": "secondary800",
    "name": "seaGreen",
    "red": 62,
    "rgb": Array [
      62,
      135,
      74,
    ],
    "rgba": Array [
      62,
      135,
      74,
      1,
    ],
    "token": "color",
    "variant": 800,
  },
  Object {
    "alpha": 1,
    "blue": 88,
    "green": 145,
    "hex": "559158",
    "hsl": Array [
      123,
      26,
      45,
    ],
    "id": "secondary700",
    "name": "hippieGreen",
    "red": 85,
    "rgb": Array [
      85,
      145,
      88,
    ],
    "rgba": Array [
      85,
      145,
      88,
      1,
    ],
    "token": "color",
    "variant": 700,
  },
  Object {
    "alpha": 1,
    "blue": 106,
    "green": 157,
    "hex": "6b9d6a",
    "hsl": Array [
      119,
      21,
      52,
    ],
    "id": "secondary600",
    "name": "aquaForest",
    "red": 107,
    "rgb": Array [
      107,
      157,
      106,
    ],
    "rgba": Array [
      107,
      157,
      106,
      1,
    ],
    "token": "color",
    "variant": 600,
  },
  Object {
    "alpha": 1,
    "blue": 124,
    "green": 169,
    "hex": "81a97c",
    "hsl": Array [
      113,
      21,
      57,
    ],
    "id": "secondary500",
    "name": "amulet",
    "red": 129,
    "rgb": Array [
      129,
      169,
      124,
    ],
    "rgba": Array [
      129,
      169,
      124,
      1,
    ],
    "token": "color",
    "variant": 500,
  },
  Object {
    "alpha": 1,
    "blue": 144,
    "green": 182,
    "hex": "97b690",
    "hsl": Array [
      109,
      21,
      64,
    ],
    "id": "secondary400",
    "name": "norway",
    "red": 151,
    "rgb": Array [
      151,
      182,
      144,
    ],
    "rgba": Array [
      151,
      182,
      144,
      1,
    ],
    "token": "color",
    "variant": 400,
  },
  Object {
    "alpha": 1,
    "blue": 186,
    "green": 210,
    "hex": "c0d2ba",
    "hsl": Array [
      105,
      21,
      78,
    ],
    "id": "secondary300",
    "name": "paleLeaf",
    "red": 192,
    "rgb": Array [
      192,
      210,
      186,
    ],
    "rgba": Array [
      192,
      210,
      186,
      1,
    ],
    "token": "color",
    "variant": 300,
  },
  Object {
    "alpha": 1,
    "blue": 208,
    "green": 224,
    "hex": "d4e0d0",
    "hsl": Array [
      105,
      21,
      85,
    ],
    "id": "secondary200",
    "name": "tasman",
    "red": 212,
    "rgb": Array [
      212,
      224,
      208,
    ],
    "rgba": Array [
      212,
      224,
      208,
      1,
    ],
    "token": "color",
    "variant": 200,
  },
  Object {
    "alpha": 1,
    "blue": 231,
    "green": 239,
    "hex": "e9efe7",
    "hsl": Array [
      105,
      20,
      92,
    ],
    "id": "secondary100",
    "name": "grayNurse",
    "red": 233,
    "rgb": Array [
      233,
      239,
      231,
    ],
    "rgba": Array [
      233,
      239,
      231,
      1,
    ],
    "token": "color",
    "variant": 100,
  },
]
`;

exports[`[mapper.js] test suite -> #mapColors: given the document colors, should transform all colors nicely 1`] = `
Array [
  Object {
    "alpha": 1,
    "blue": 159,
    "green": 6,
    "hex": "10069f",
    "hsl": Array [
      244,
      93,
      32,
    ],
    "name": "ultramarine",
    "red": 16,
    "rgb": Array [
      16,
      6,
      159,
    ],
    "rgba": Array [
      16,
      6,
      159,
      1,
    ],
    "token": "color",
  },
  Object {
    "alpha": 1,
    "blue": 80,
    "green": 0,
    "hex": "e50050",
    "hsl": Array [
      339,
      100,
      45,
    ],
    "name": "amaranth",
    "red": 229,
    "rgb": Array [
      229,
      0,
      80,
    ],
    "rgba": Array [
      229,
      0,
      80,
      1,
    ],
    "token": "color",
  },
  Object {
    "alpha": 1,
    "blue": 78,
    "green": 190,
    "hex": "2cbe4e",
    "hsl": Array [
      134,
      62,
      46,
    ],
    "name": "apple",
    "red": 44,
    "rgb": Array [
      44,
      190,
      78,
    ],
    "rgba": Array [
      44,
      190,
      78,
      1,
    ],
    "token": "color",
  },
  Object {
    "alpha": 1,
    "blue": 0,
    "green": 0,
    "hex": "000000",
    "hsl": Array [
      0,
      0,
      0,
    ],
    "name": "black",
    "red": 0,
    "rgb": Array [
      0,
      0,
      0,
    ],
    "rgba": Array [
      0,
      0,
      0,
      1,
    ],
    "token": "color",
  },
]
`;

exports[`[mapper.js] test suite -> #mapGradients: given artboard gradients, should transform them nicely 1`] = `
Array [
  Object {
    "from": Array [
      0.5,
      1,
    ],
    "id": "linear",
    "stops": Array [
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 144,
          "green": 182,
          "hex": "97b690",
          "hsl": Array [
            109,
            21,
            64,
          ],
          "name": "norway",
          "red": 151,
          "rgb": Array [
            151,
            182,
            144,
          ],
          "rgba": Array [
            151,
            182,
            144,
            1,
          ],
          "token": "color",
        },
        "position": 0,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 88,
          "green": 145,
          "hex": "559158",
          "hsl": Array [
            123,
            26,
            45,
          ],
          "name": "hippieGreen",
          "red": 85,
          "rgb": Array [
            85,
            145,
            88,
          ],
          "rgba": Array [
            85,
            145,
            88,
            1,
          ],
          "token": "color",
        },
        "position": 0.5038507579143953,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 62,
          "green": 125,
          "hex": "227d3e",
          "hsl": Array [
            138,
            57,
            31,
          ],
          "name": "salem",
          "red": 34,
          "rgb": Array [
            34,
            125,
            62,
          ],
          "rgba": Array [
            34,
            125,
            62,
            1,
          ],
          "token": "color",
        },
        "position": 1,
      },
    ],
    "to": Array [
      0.4999999999999999,
      3.0616169978683836e-17,
    ],
    "token": "gradient",
    "type": "LINEAR",
  },
  Object {
    "from": Array [
      0.5,
      1,
    ],
    "id": "radial",
    "stops": Array [
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 144,
          "green": 182,
          "hex": "97b690",
          "hsl": Array [
            109,
            21,
            64,
          ],
          "name": "norway",
          "red": 151,
          "rgb": Array [
            151,
            182,
            144,
          ],
          "rgba": Array [
            151,
            182,
            144,
            1,
          ],
          "token": "color",
        },
        "position": 0,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 62,
          "green": 125,
          "hex": "227d3e",
          "hsl": Array [
            138,
            57,
            31,
          ],
          "name": "salem",
          "red": 34,
          "rgb": Array [
            34,
            125,
            62,
          ],
          "rgba": Array [
            34,
            125,
            62,
            1,
          ],
          "token": "color",
        },
        "position": 1,
      },
    ],
    "to": Array [
      0.5,
      -0.27263153394819584,
    ],
    "token": "gradient",
    "type": "RADIAL",
  },
  Object {
    "from": Array [
      0.5,
      1,
    ],
    "id": "angular",
    "stops": Array [
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 144,
          "green": 182,
          "hex": "97b690",
          "hsl": Array [
            109,
            21,
            64,
          ],
          "name": "norway",
          "red": 151,
          "rgb": Array [
            151,
            182,
            144,
          ],
          "rgba": Array [
            151,
            182,
            144,
            1,
          ],
          "token": "color",
        },
        "position": 0.4336458341191861,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 62,
          "green": 125,
          "hex": "227d3e",
          "hsl": Array [
            138,
            57,
            31,
          ],
          "name": "salem",
          "red": 34,
          "rgb": Array [
            34,
            125,
            62,
          ],
          "rgba": Array [
            34,
            125,
            62,
            1,
          ],
          "token": "color",
        },
        "position": 1,
      },
    ],
    "to": Array [
      0.5,
      -0.27263153394819584,
    ],
    "token": "gradient",
    "type": "ANGULAR",
  },
]
`;

exports[`[mapper.js] test suite -> #mapGradients: given docuement gradients, should transform them nicely 1`] = `
Array [
  Object {
    "from": Array [
      0.5,
      1,
    ],
    "stops": Array [
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 144,
          "green": 182,
          "hex": "97b690",
          "hsl": Array [
            109,
            21,
            64,
          ],
          "name": "norway",
          "red": 151,
          "rgb": Array [
            151,
            182,
            144,
          ],
          "rgba": Array [
            151,
            182,
            144,
            1,
          ],
          "token": "color",
        },
        "position": 0,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 88,
          "green": 145,
          "hex": "559158",
          "hsl": Array [
            123,
            26,
            45,
          ],
          "name": "hippieGreen",
          "red": 85,
          "rgb": Array [
            85,
            145,
            88,
          ],
          "rgba": Array [
            85,
            145,
            88,
            1,
          ],
          "token": "color",
        },
        "position": 0.5038507579143953,
      },
      Object {
        "color": Object {
          "alpha": 1,
          "blue": 62,
          "green": 125,
          "hex": "227d3e",
          "hsl": Array [
            138,
            57,
            31,
          ],
          "name": "salem",
          "red": 34,
          "rgb": Array [
            34,
            125,
            62,
          ],
          "rgba": Array [
            34,
            125,
            62,
            1,
          ],
          "token": "color",
        },
        "position": 1,
      },
    ],
    "to": Array [
      0.4999999999999999,
      3.0616169978683836e-17,
    ],
    "token": "gradient",
    "type": "LINEAR",
  },
]
`;

exports[`[mapper.js] test suite -> #mapShadows: given artboard gradients, should transform them nicely 1`] = `
Array [
  Object {
    "blur": 3,
    "color": Object {
      "alpha": 0.5,
      "blue": 0,
      "green": 0,
      "hex": "000000",
      "hsl": Array [
        0,
        0,
        0,
      ],
      "name": "black",
      "red": 0,
      "rgb": Array [
        0,
        0,
        0,
      ],
      "rgba": Array [
        0,
        0,
        0,
        0.5,
      ],
      "token": "color",
    },
    "id": "simple",
    "spread": 0,
    "token": "shadow",
    "type": "inner-shadow",
    "x": 0,
    "y": 1,
  },
  Object {
    "blur": 4,
    "color": Object {
      "alpha": 0.5,
      "blue": 0,
      "green": 0,
      "hex": "000000",
      "hsl": Array [
        0,
        0,
        0,
      ],
      "name": "black",
      "red": 0,
      "rgb": Array [
        0,
        0,
        0,
      ],
      "rgba": Array [
        0,
        0,
        0,
        0.5,
      ],
      "token": "color",
    },
    "id": "simple",
    "spread": 0,
    "token": "shadow",
    "type": "drop-shadow",
    "x": 0,
    "y": 2,
  },
]
`;

exports[`[mapper.js] test suite -> #mapTextStyles: given the text styles, should transform them nicely 1`] = `
Array [
  Object {
    "borderBottom": null,
    "color": Object {
      "alpha": 1,
      "blue": 159,
      "green": 6,
      "hex": "10069f",
      "hsl": Array [
        244,
        93,
        32,
      ],
      "name": "ultramarine",
      "red": 16,
      "rgb": Array [
        16,
        6,
        159,
      ],
      "rgba": Array [
        16,
        6,
        159,
        1,
      ],
      "token": "color",
    },
    "family": "FiraSans-Regular",
    "id": "heading1",
    "kerning": null,
    "size": 48,
    "token": "text-style",
    "weight": null,
  },
  Object {
    "borderBottom": null,
    "color": Object {
      "alpha": 1,
      "blue": 159,
      "green": 6,
      "hex": "10069f",
      "hsl": Array [
        244,
        93,
        32,
      ],
      "name": "ultramarine",
      "red": 16,
      "rgb": Array [
        16,
        6,
        159,
      ],
      "rgba": Array [
        16,
        6,
        159,
        1,
      ],
      "token": "color",
    },
    "family": "FiraSans-Regular",
    "id": "heading2",
    "kerning": null,
    "size": 36,
    "token": "text-style",
    "weight": null,
  },
  Object {
    "borderBottom": null,
    "color": Object {
      "alpha": 1,
      "blue": 0,
      "green": 0,
      "hex": "000000",
      "hsl": Array [
        0,
        0,
        0,
      ],
      "name": "black",
      "red": 0,
      "rgb": Array [
        0,
        0,
        0,
      ],
      "rgba": Array [
        0,
        0,
        0,
        1,
      ],
      "token": "color",
    },
    "family": "FiraSans-Regular",
    "id": "paragraph",
    "kerning": null,
    "size": 18,
    "token": "text-style",
    "weight": null,
  },
]
`;
